src/components/touchables/GenericTouchable.tsx

```diff
@@ -66,6 +67,16 @@ export default class GenericTouchable extends Component<
   // State of touchable
   STATE: TouchableState = TOUCHABLE_STATE.UNDETERMINED;
 
+  constructor(props: GenericTouchableProps & InternalProps) {
+    super(props);
+
+    console.warn(
+      tagMessage(
+        'TouchableOpacity component will be removed in Gesture Handler 4'
+      )
+    );
```

Comment:
Please don't, we've been there already - https://github.com/software-mansion/react-native-gesture-handler/issues/1831

Reply:
Okay, you've convinced me

Reply:
Done in [addb507](https://github.com/software-mansion/react-native-gesture-handler/pull/3260/commits/addb5075f09fa1eaeade838b12beb8b192d3d20a)